<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableWindow</class>
 <widget class="QMainWindow" name="TableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>431</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正粗金陵简体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>用户</string>
     </property>
     <property name="font">
      <font>
       <family>方正粗金陵简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>密码</string>
     </property>
     <property name="font">
      <font>
       <family>方正粗金陵简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>可用余额</string>
     </property>
     <property name="font">
      <font>
       <family>方正粗金陵简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>等级</string>
     </property>
     <property name="font">
      <font>
       <family>方正粗金陵简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="renovate_button">
       <property name="font">
        <font>
         <family>方正粗金陵简体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_user_button">
       <property name="font">
        <font>
         <family>方正粗金陵简体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除用户</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="change_image_button">
       <property name="font">
        <font>
         <family>方正粗金陵简体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>更换背景图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="change_music_button">
       <property name="font">
        <font>
         <family>方正粗金陵简体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>更换背景音乐</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
